interface-definitions/system-event-handler.xml.in

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interfaceDefinition>
+  <node name="system">
```

Comment:
Maybe switch to `service event-handler`? I can't remember why exactly I put it in `system` in the old implementation. What do you think?

Reply:
I have nothing against it. I am for any option.

Reply:
@dmbaturin Done
```
set service event-handler first pattern '.*ssh2.*'
set service event-handler first script '/config/scripts/hello.sh'

```